From caf357aecf85916dc605e513e0d3e6765479b172 Mon Sep 17 00:00:00 2001
From: Michal Marek <mmarek@suse.cz>
Date: Fri, 3 Jun 2011 12:45:12 +0200
Subject: [PATCH] scripts/kmsg-doc: Do not require Switch.pm
Patch-mainline: depends on patches.arch/s390-message-catalog.diff

It is not used by the script and the module is no longer available in
the perl core.

Signed-off-by: Michal Marek <mmarek@suse.cz>

diff --git a/scripts/kmsg-doc b/scripts/kmsg-doc
index 07a0a2e..e0f64ed 100644
--- a/scripts/kmsg-doc
+++ b/scripts/kmsg-doc
@@ -14,7 +14,6 @@
 #
 
 use Cwd;
-use Switch;
 use bigint;
 
 my $errors = 0;
